src/index.html

```diff
@@ -1,159 +1,169 @@
 <!DOCTYPE html>
 <html lang="en">
-
-<head>
-  <meta charset="utf-8">
-  <link rel="stylesheet" type="text/css" href="./styles.css">
-  <link href="https://fonts.googleapis.com/css2?family=Raleway&family=Shadows+Into+Light&display=swap" rel="stylesheet">
-  <link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.1/css/all.min.css" rel="stylesheet">
-  <title>Activity Tracker</title>
-</head>
-
-<body>
-  <form>
-    <label for="names">Choose a user</label>
-    <select name="names-list" class="names-list" id="names"></select>
-    <label for="date">Choose a Date</label>
-    <input type="date" class="date-picker" id="date" value="2019-06-15" min="2019-06-15" max="2019-09-22">
-    <button type="submit" class="admin-selector">View Data</button>
-  </form>
-  <main>
-  <section>
-    <header>
-      <h1 class="user-first-name">User Name</h1>
-      <div class="user-info-text">
-        <p class="user-info user-address">Address</p>
-        <p class="user-info user-email">Email</p>
-        <p class="user-info user-step-compare">Steps</p>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="./styles.css">
+    <link href="https://fonts.googleapis.com/css2?family=Raleway&family=Shadows+Into+Light&display=swap" rel="stylesheet">
+    <link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.1/css/all.min.css" rel="stylesheet">
+    <title>Activity Tracker</title>
+  </head>
+  <body>
+    <form>
+      <div class="chooser">
+        <label for="names">Choose a user</label>
+        <select name="names-list" class="names-list" id="names"></select>
       </div>
-    </header>
-      <!-- <span>
-        <p class="admin-text">Admin</p>
-        <img src="../assets/admin.svg" class="admin-icon">
-      </span> -->
-    </section>
-    <section class="daily-stats">
-      <h2>View Daily Stats</h2>
-      <article class="daily-data">
-
-        <section class="steps current-stats">
-          <div class="daily-stat-icons">
-            <img src="../assets/person.svg" class="exercise-icon icon">
-            <p class="stats-text">steps</p>
-          </div>
-        </section>
-
-
-        <section class="sleep-button current-stats">
-          <div class="daily-stat-icons">
-            <img src="../assets/sleep.svg" class="sleep-icon icon">
-            <p class="stats-text">sleep</p>
-          </div>
-        </section>
-        <section class="hydration current-stats">
-          <div class="daily-stat-icons">
-            <img src="../assets/water.svg" class="water-icon icon">
-            <p class="stats-text">water</p>
+      <div class="chooser">
+        <label for="date">Choose a start date</label>
+        <input type="date" class="date-picker" id="date" value="2019-06-15" min="2019-06-15" max="2019-09-22">
```

Comment:
Are these min and max values intended?

Reply:
Yes, we only have data for between those dates.

Reply:
In the real world, you'd be able to select dates starting with the first day you used the app, up until the current day.